CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 ### Released packages
 
 - xo-server-backup-reports v0.16.1
+- @marsaud/smb2 v0.14.0
```

Comment:
This package is a dependency which has already been released and should not be listed in our changelog.

But you need to upgrade this dep in `@xen-orchestra/fs`'s `package.json`.